package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "rover-basestation-frontend",
+	"version": "0.0.1",
+	"type": "module",
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
```

Comment:
make sure the `build` command works properly and places the built files in the output directory

Reply:
Configuration performed in svelte.config.js.